test/permissionsMockData.js

```diff
@@ -0,0 +1,25 @@
+const validArrayInputs = [
```

Comment:
add also long input (at least 100 length) - to make sure it can work with large numbers